src/webserver.js

```diff
@@ -103,9 +103,9 @@ function initializeNodeBB(callback) {
 		},
 		function (next) {
 			async.series([
-				async.apply(meta.sounds.init),
-				async.apply(languages.init),
-				async.apply(meta.blacklist.load)
+				meta.sounds.addUploads,
+				languages.init,
+				meta.blacklist.load,
```

Comment:
Trailing comma here. I bet you'll make the argument that future additions will mean this line won't be changed, but this goes against style guide™